onnx/defs/shape_inference.h

```diff
@@ -322,4 +322,50 @@ inline void propagateShapeFromAttributeToOutput(
   updateOutputShape(ctx, outputIndex, shape);
 }
 
+inline void bidirectionalBroadcastShapeInference(
```

Comment:
Call it numpyBroadcast?

Reply:
I think we won't have any function called `unidirectionalBroadcastShapeInference`, so calling it `numpyBroadcasting` is fine.